supertokens_python/recipe/totp/asyncio/__init__.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License") as published by the Apache Software Foundation.
+#
+# You may not use this file except in compliance with the License. You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+from typing import Optional, Union, Dict, Any
+from ..interfaces import (
+    CreateDeviceOkResult,
+    VerifyDeviceOkResult,
+    VerifyCodeOkResult,
+    ListTotpDevicesOkResult,
+    RemoveTotpDeviceOkResult,
+)
+from ..interfaces import (
+    DeviceAlreadyExistsError,
+    InvalidTotpError,
+    TotpNotEnabledError,
+    LimitReachedError,
+    UnknownDeviceError,
+)
+
+
+async def create_totp_device(
+    user_id: str,
+    device_name: str,
+    skew: Optional[int] = None,
+    period: Optional[int] = None,
+    user_context: Optional[Dict[str, Any]] = None,
+) -> Union[CreateDeviceOkResult, DeviceAlreadyExistsError]:
+    ...
+
+
+async def verify_code(
+    user_id: str, code: int, user_context: Optional[Dict[str, Any]] = None
+) -> Union[
+    VerifyDeviceOkResult, InvalidTotpError, TotpNotEnabledError, LimitReachedError
```

Comment:
TotpNotEnabledError -> does this mean the license key is not put? Cause that results in a 4xx error to be thrown from the core.

Reply:
If it does not mean the license key is missing, then when is this error thrown?